repos/system_upgrade/el7toel8/actors/checkkdegnome/libraries/library.py

```diff
@@ -0,0 +1,158 @@
+import os
+import subprocess
+import time
+
+from leapp import reporting
+from leapp.libraries.stdlib import api
+
+ONE_MONTH = 2592000  # Number of seconds in one month
+
+
+def is_executable(path):
+    """
+    Checks if path exists, if it is file and if is executable.
+    """
+    return os.path.exists(path) and os.path.isfile(path) and os.access(path, os.X_OK)
+
+
+def get_xsession(path):
+    """
+    Gets current XSession. If there is more than one definition of xsession for some reason,
+     function returns last definition. Return empty string if no XSession found in file given
+     by path (any reason including bad path etc.)
```

Comment:
The format again. Do not add addiotional empty space on other lines, I mean, do not do this
```
Something
 something
```

Reply:
This is strange, I don't know how the extra space got there. Fixed, of course.